### PR TITLE
Del the unnecessary buffer

### LINKED ISSUE #4: When I do "Vim . ", a buffer remains
Thank you for the great plugin!:pray:

when I do "vim .", abuffer remains.
Is this the intended behavior?

## GIF
[Image from Gyazo](https://gyazo.com/b9b19155bf0bd028a9c8c1bde2c39419)

## Structure
* vim 8.1
```vim
call plug#begin('~/.vim/autoload')
Plug 'lambdalisue/fern.vim'
Plug 'lambdalisue/fern-hijack.vim'
call plug#end()
```

### PR DESCRIPTION
**Pull request by DuGlaser**
resolve #4

### CODE REVIEW

**Comment by lambdalisue** [comment]
Thanks for your contribution. I'd like to know what happens with the following commands

```
:e README.md
:e LICENSE
:e .
<C-o>
```

The expected behavior of above is to back `LICENSE`

Rel. https://github.com/lambdalisue/fern-hijack.vim/issues/2

**Comment by DuGlaser** [comment]
It's back to README.md...

**Comment by DuGlaser** [comment]
The results for both ``vim``  and  ``neovim``  were as follows.
```
:e README.md
:e LICENSE
:e .
<C-o>

It's back to README.md...
```
```
:e README.md
:e LICENSE
:Fern .
<C-o>

It's back to LICENSE!
```

File: plugin/fern_hijack.vim [hunk 1]
```diff
@@ -8,6 +8,7 @@ function! s:hijack_directory() abort
   if !isdirectory(path)
     return
   endif
+  bwipeout %
```
Comment (lambdalisue) [inline plugin/fern_hijack.vim]: Please add `keepjumps` and `keepalt` like

```suggestion
  keepjumps keepalt bwipeout %
```

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: plugin/fern_hijack.vim (hunk 1 above)
Comment (DuGlaser) [reply plugin/fern_hijack.vim]: Thank you! It's working ``vim`` and ``NeoVim``!
```
:e README.md
:e LICENSE
:e .
<C-o>

It's back to LICENSE!
```

**Comment by lambdalisue** [comment]
Merged. Thanks 👍

**Pull request merged**